docs/rst/user_manual/configuration.rst

```diff
@@ -40,7 +36,7 @@ with future releases.
         - ``v3.0``
         - *v0.3.0*
 
-**Current configuration version is v3.0**.
+**Current configuration version is v4.0**.
```

Comment:
This section needs some more updating.

Reply:
Also, all YAML examples need to be updated (sorry) and it should be enforced that the user only specifies 4.0 version o no version (LATEST=4.0).

Reply:
Please fix meaningless substitutions.